config/environments/development.rb

```diff
@@ -19,6 +19,9 @@
 
   config.asset_host = config.three_scale.asset_host.presence
 
+  # Match custom domains on development
+  config.hosts << /[a-z0-9-]+\.3scale\.localhost/
```

Comment:
The default stopped working after the update? Either case, I'd suggest that we allow anything `.*\.localhost`

Reply:
It sees we need it. I tried removing it and porta doesn't load. I changed the regexp to `/.+\.localhost/`